src/Features/CSharp/Portable/ConvertIfToSwitch/CSharpConvertIfToSwitchCodeRefactoringProvider.cs

```diff
@@ -1,277 +1,30 @@
-                                    when p.Designation is SingleVariableDesignationSyntax designation:
-                                    return new Pattern.ByType(p.Type, designation.Identifier);
-
-                                case ConstantPatternSyntax p:
-                                    return new Pattern.ByValue(p.Expression);
-
-                                default:
-                                    return null;
-                            }
-                        }
-
-                    case SyntaxKind.LogicalAndExpression:
-                        {
-                            // Look for the form "expr && cond" where "expr" must be one of the above cases.
-                            // This will turn into a case guard e.g. "case pat when cond:".
-                            var node = (BinaryExpressionSyntax)operand;
-
-                            // Since "&&" operator has a left associativity, first we need to find the leftmost
-                            // expression in the if-condition, then we can replace it with an appropriate pattern.
-                            var leftmost = GetLeftmostCondition(node);
-
-                            // Make sure that it is one of the expressions that we can handle to prevent recursion.
-                            switch (leftmost.Kind())
-                            {
-                                case SyntaxKind.IsPatternExpression:
-                                case SyntaxKind.IsExpression:
-                                case SyntaxKind.EqualsExpression:
-                                    break;
-
-                                default:
-                                    return null;
-                            }
-
-                            var pattern = CreatePatternFromExpression(leftmost);
-                            if (pattern == null)
-                            {
-                                return null;
-                            }
-
-                            // In case of chained "&&" operators, we need to justify the precedence of operands in which
-                            // the "&&" operator is applied. For example, in "if (expr && cond1 && cond2)" we need to
-                            // separate out the leftmost expression and reconstruct the rest of conditions into a new
-                            // expression, so that they would be evaluated entirely after "expr" e.g. "expr && (cond1 && cond2)".
-                            // Afterwards, we can directly use it in a case guard e.g. "case pat when cond1 && cond2:".
-                            var parentBinary = (BinaryExpressionSyntax)leftmost.WalkUpParentheses().Parent;
-                            var grandparentBinary = parentBinary.WalkUpParentheses().Parent as BinaryExpressionSyntax;
+        public override string GetTitle(bool forSwitchExpression)
+            => forSwitchExpression
+                ? "TODO"
```

Comment:
?

Reply:
I postpone this so I don't need to deal with resource changes through possible merge conflicts - it usually takes a long time to reach there.